src/osd/OSD.cc

```diff
@@ -7630,7 +7637,7 @@ void OSD::sched_scrub()
       if (pg->m_scrubber->is_reserving()) {
 	pg->unlock();
 	dout(10) << __func__ << ": reserve in progress pgid " << scrub_job.pgid << dendl;
-	break;
+	goto out;
```

Comment:
why 'goto' instead of the 'break'?

Reply:
We don't _break_ here because I don't want to clear the reserve_failed flags from all PGs.